app/protected/page.tsx

```diff
@@ -1,18 +1,176 @@
-import SignOut from "@/components/sign-out";
+"use client";
```

Comment:
This was my first time working with NextJS 13 and the app directory. I am used to the pages setup. So I did not have experience yet with React Server Components. I stuck with using a client component here because it is what I am used to developing with in React but I plan to learn more about when SSR is preferable to client side rendering.